apps/deploy-web/src/utils/templates.ts

```diff
@@ -74,4 +74,50 @@ deployment:
 `
 };
 
-export const hardcodedTemplates = [sdlBuilderTemplate, helloWorldTemplate];
+export const github = {
+  title: "GitHub",
+  name: "GitHub",
+  code: "github",
+  category: "General",
+  description: "Get started with a simple linux Ubuntu server!",
+  githubUrl: "",
+  valuesToChange: [],
+  content: `---
```

Comment:
Maybe we could put this template in the awesome akash repo, so when you need to update it we don't need to release a new version of the console.

Reply:
Done, now we are fetching this from awesome akash repo.

Reply:
So we can remove it from here?